The linux builds for "cctz", "clblas", "discord-rpc", "matroska" all fail with the same message which seems unrelated to this PR:

```
/usr/include/pthread.h:399:12: error: ‘cpu_set_t’ does not name a type
      const cpu_set_t *__cpuset)
            ^~~~~~~~~
```

--- a/ports/gsl-lite/portfile.cmake
+++ b/ports/gsl-lite/portfile.cmake
@@ -12,13 +12,7 @@ vcpkg_configure_cmake(
 )
 
 vcpkg_install_cmake()
-
-vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/gsl-lite" TARGET_PATH "share/gsl-lite")
-
-file(REMOVE_RECURSE
-    "${CURRENT_PACKAGES_DIR}/lib"
-    "${CURRENT_PACKAGES_DIR}/debug"
-)
+vcpkg_fixup_cmake_targets(CONFIG_PATH "lib/cmake/gsl-lite")
 
 file(WRITE ${CURRENT_PACKAGES_DIR}/include/gsl-lite.hpp "#ifndef GSL_LITE_HPP_VCPKG_COMPAT_HEADER_INCLUDED
 #define GSL_LITE_HPP_VCPKG_COMPAT_HEADER_INCLUDED
@@ -26,7 +20,9 @@ file(WRITE ${CURRENT_PACKAGES_DIR}/include/gsl-lite.hpp "#ifndef GSL_LITE_HPP_VC
 #include <gsl/gsl-lite.hpp>
 #endif // GSL_LITE_HPP_VCPKG_COMPAT_HEADER_INCLUDED")
 
-file(
-    INSTALL "${SOURCE_PATH}/LICENSE"
-    DESTINATION "${CURRENT_PACKAGES_DIR}/share/gsl-lite"
-    RENAME copyright)
+file(REMOVE_RECURSE
+    "${CURRENT_PACKAGES_DIR}/lib"
+    "${CURRENT_PACKAGES_DIR}/debug"
+)
+
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

